### PR TITLE
Use `student@os` consistently as part of the prompt in code snippets

### PR DESCRIPTION
**Pull request by razvand**
Use `student@os` consistently as part of the prompt in code snippets.

### CODE REVIEW

**Review by teodutu — APPROVED** [review]

File: content/chapters/io/lecture/demo/optimizations/README.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ See [server.py](#serverpy).
 This is a simple bash script that receives the port of a running server as argument and proceeds to run multiple **client.py** instances.
```
Comment (teodutu) [inline content/chapters/io/lecture/demo/optimizations/README.md:156]:
```suggestion
This is a simple bash script that receives the port of a running server as argument and proceeds to run multiple `client.py` instances.
```
To pacify the linter.

**Review by teodutu — CHANGES_REQUESTED** [review]
I approved this by mistake. The suggestion I made will only fix one linter error. `MD041` will be fixed by another PR. I created an issue about it: https://github.com/open-education-hub/operating-systems/issues/179.

File: content/chapters/io/lecture/demo/optimizations/README.md (hunk 1 above)
Comment (razvand) [reply content/chapters/io/lecture/demo/optimizations/README.md]: @teodutu, did.

**Review by teodutu — APPROVED** [review]
I'm OK with this. @razvand you decide if/how you want to squash the commits before merging.

**Review by Alex-deVis — APPROVED** [review]
@razvand, @teodutu, I am okay with merging it without the fix for `MD041`. We can address that in another PR.

**Comment by razvand** [comment]
@Alex-deVis, @teodutu, I'm left only the `student@os` commits and I squashed them into a single one.

**Pull request merged**